### PR TITLE
Add sanity check for liveness

### PR DESCRIPTION
**Pull request by maciekzygmunt**
Resolves L2B-3163

### CODE REVIEW

File: packages/backend/src/modules/liveness/LivenessModule.ts [hunk 1]
```diff
@@ -54,6 +54,8 @@ export function createLivenessModule(
   const livenessController = new LivenessController(
     livenessRepository,
     config.projects,
+    clock,
+    livenessConfigurationRepository,
```
Comment (antooni) [inline packages/backend/src/modules/liveness/LivenessModule.ts:58]: pass logger and send error to sentry when sanity check fails (.errror() method call)

Comment (maciekzygmunt) [reply packages/backend/src/modules/liveness/LivenessModule.ts:58]: [eb0acbf](https://github.com/l2beat/l2beat/pull/2267/commits/eb0acbf5cf87858018c365659a6d441d7db67f3c)

File: packages/backend/src/api/controllers/liveness/LivenessController.ts [hunk 2]
```diff
@@ -1,26 +1,54 @@
...
 
+    const requiredTimestamp = this.clock.getLastHour().add(-1, 'hours')
+    const configurations = await this.configurationRepository.getAll()
+    const areAllSynced = configurations.every((config) =>
+      config.lastSyncedTimestamp?.gte(requiredTimestamp),
+    )
+
+    if (!areAllSynced) {
+      console.timeEnd('getLiveness')
+      this.logger.error('[LIVENESS]: DATA_NOT_FULLY_SYNCED')
```
Comment (antooni) [inline packages/backend/src/api/controllers/liveness/LivenessController.ts:48]:
```suggestion
      this.logger.error('[API]: Liveness data is not synced')
```

File: packages/backend/src/api/controllers/liveness/LivenessController.ts [hunk 3]
```diff
@@ -1,26 +1,54 @@
...
+    const requiredTimestamp = this.clock.getLastHour().add(-1, 'hours')
+    const configurations = await this.configurationRepository.getAll()
+    const areAllSynced = configurations.every((config) =>
+      config.lastSyncedTimestamp?.gte(requiredTimestamp),
+    )
+
+    if (!areAllSynced) {
+      console.timeEnd('getLiveness')
+      this.logger.error('[API]: Liveness data is not synced')
+      return { type: 'error', error: 'DATA_NOT_FULLY_SYNCED' }
```
Comment (antooni) [inline packages/backend/src/api/controllers/liveness/LivenessController.ts:49]:
```suggestion
      return { type: 'error', error: 'DATA_NOT_SYNCED' }
```

**Review by antooni — APPROVED** [review]

**Pull request merged**